pcdsdevices/epics_motor.py

```diff
@@ -1593,7 +1609,9 @@ def _GetMotorClass(basepv):
                    ('MMB', BeckhoffAxis),
                    ('PIC', PCDSMotorBase),
                    ('MCS', SmarAct),
-                   ('MCS2', SmarAct))
+                   ('MCS2', SmarAct),
+                   ('HEX', PI_M824))
```

Comment:
I'm looking through the PV lists to see if any non-hexapod motor has HEX in it.
If this looks good there's no reason not to approve this PR

Reply:
All instances of `:HEX:` currently are these MEC hexapods :+1: